src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -1730,18 +1730,39 @@ static public String extractEmailFromToHeader(String header) {
 		 */
 
 		protected JSONObject createGoogleGroupForSite(String sessionId, String siteId) {
-			String server = env.getProperty(Utils.GGB_SERVER_NAME);
+
+			GGBApiWrapper ggb = establishGGBConnection();
 
 			JSONObject googleGroupSettings = getGoogleGroupSettings(sessionId, siteId);
 
-			GGBApiWrapper ggb = new GGBApiWrapper(server,null);
-			String url = String.format("/groups/%s",googleGroupSettings.getString("email"));
-			HttpResponse ggb_response = ggb.put_request(url, googleGroupSettings.toString());
-			// TODO: error checking.
+			String new_group_url = String.format("/groups/%s",googleGroupSettings.getString("email"));
+			ApiResultWrapper arw = ggb.put_request(new_group_url,googleGroupSettings.toString());
+
```

Comment:
Here it `ggb.put_request` is returning the ApiResultWrapper Object, which is good but not consistence with Post request